packages/components/package.json

```diff
@@ -23,6 +23,7 @@
   "module": "dist/esm/index.mjs",
   "types": "dist/types/index.d.ts",
   "sideEffects": [
+    "tailwind.css",
```

Comment:
Actually how come this is necessary? Consumers don't need to import this, and it seems like the components can access it fine

Reply:
Tested in unified-home and it was tree-shaken lol

Reply:
Ohh, this key needs a better name lol